packages/nodes-base/nodes/Medium/Medium.node.ts

```diff
@@ -0,0 +1,448 @@
+import {
+	IExecuteFunctions,
+} from 'n8n-core';
+
+import {
+	IDataObject,
+	ILoadOptionsFunctions,
+	INodeExecutionData,
+	INodeType,
+	INodePropertyOptions,
+	INodeTypeDescription,
+} from 'n8n-workflow';
+
+import {
+	mediumApiRequest,
+} from './GenericFunctions';
+
+export class Medium implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'Medium',
+		name: 'medium',
+		group: ['output'],
+		icon: 'file:medium.png',
+		version: 1,
+		description: 'Consume Medium API',
+		subtitle: '={{$parameter["operation"] + ": " + $parameter["resource"]}}',
+		defaults: {
+			name: 'Medium',
+			color: '#772244',
+		},
+		inputs: ['main'],
+		outputs: ['main'],
+		credentials: [
+			{
+				name: 'mediumApi',
+				required: true,
+			}
+		],
+		properties: [
+			{
+				displayName: 'Authentication',
+				name: 'authentication',
+				type: 'options',
+				options: [
+					{
+						name: 'Access Token',
+						value: 'accessToken',
+					},
+					// {
+					// 	name: 'OAuth2',
+					// 	value: 'oAuth2',
+					// },
+
+				],
+				default: 'accessToken',
+				description: 'The method of authentication.',
+			},
+			{
+				displayName: 'Resource',
+				name: 'resource',
+				type: 'options',
+				options: [
+					{
+						name: 'Post',
+						value: 'post',
+					},
+
+				],
+				default: 'post',
+				description: 'Resource to consume.',
+			},
+			{
+				displayName: 'Operation',
+				name: 'operation',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: [
+							'post',
+						],
+					},
+				},
+				options: [
+					{
+						name: 'Create a post',
+						value: 'create',
+						description: 'Create a post on the user authenticated profile',
+					},
+
+				],
+				default: 'create',
+				description: 'The operation to perform.',
+			},
+
+			// ----------------------------------
+			//         post:create
+			// ----------------------------------
+			{
+				displayName: 'Publication',
+				name: 'publication',
+				type: 'boolean',
+				default: false,
+				description: 'Are you publishing under a publication?'
```

Comment:
I've never posted on Medium so I'm not sure, but what about `'Are you posting for a publication?'`?

Reply:
Also, I've never seen a boolean setting for a node phrased as a question. Please double check.